eos/fio/frontend/src/components/MapHandle.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import "./common.css";
+import { message, Spin } from "antd";
+import { FioWallet } from "fio-wallet";
```

Comment:
How is the frontend using the lib package? It has not been installed.

Reply:
Installing using yalc